Service/src/routes/charts.py

```diff
@@ -107,6 +122,8 @@ def formatDataIntraday(jsonData):
          'epochTime' : itemTime.timestamp(),
          'price' : item['average']
       })
+   # if NoneItemsIEX > 2:
```

Comment:
no commented code, this is a code smell, either remove it or make it work

Reply:
Removed, and fixed the code that was leading to that exception... I had to filter out the improper output items from IEX itself